### PR TITLE
Create v6.0-performance-benchmarking-with-tpch.md

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by TomShawn — APPROVED** [review]

File: benchmark/v6.0-performance-benchmarking-with-tpch.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: TiDB TPC-H 性能对比测试报告 - v6.0 MPP 模式对比 Greenplum 6.15.0 以及 Apache Spark 3.1.1
```
Comment (moyun) [inline benchmark/v6.0-performance-benchmarking-with-tpch.md:2]: 我们稍微改一下吧，因为下文说的是不提供对比，这个title却写的是“6.0 mpp 模式对比”，恐怕会有误解了。
直接写成： `TiFlash 与 Greenplum, Spark 性能比较`
应该比较妥当。

File: benchmark/v6.0-performance-benchmarking-with-tpch.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+---
+
+# TiDB TPC-H 性能对比测试报告
+
+TiDB v6.0 暂不提供 MPP 模式的性能测试报告，与其他 MPP 系统和计算框架（Greeplum，Spark）的性能对比，请参考 [TiDB v5.4 TPC-H 性能对比测试报告](/benchmark/v5.4-performance-benchmarking-with-tpch.md)。
```
Comment (moyun) [inline benchmark/v6.0-performance-benchmarking-with-tpch.md:7]: 意思没有问题，咱们精炼一下：
`请参考 [TiDB v5.4 TPC-H 性能对比测试报告](/benchmark/v5.4-performance-benchmarking-with-tpch.md)。`

**Review by moyun — COMMENTED** [review]
再稍微调整下是否更好？

**Review by moyun — APPROVED** [review]
LGTM

**Pull request merged**